content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins.adoc

```diff
@@ -0,0 +1,1525 @@
+It means just more work for you._
+
+[[JenkinsBehindProxy-HowJenkinshandlesProxyServers]]
+==== How Jenkins handles Proxy Servers
+
+You can configure the proxy server that Jenkins will use by going to
+*Manage Jenkins* > *Manage Plugins* > *Advanced*. 
+This is preferred over setting JVM properties.
+
+Here's a screenshot showing an example configuration:
+
+image::proxy-configuration.png["Proxy Configuration"]
+
+[cols=",",]
+|===
+|*Field* |*Description*
+
+|Server |Host name or address of the proxy server. Example:
+`+proxy.example.com+`
+
+|Port |The port the proxy server listens on. Example: `+8080+`
+
+|User name |If the proxy server requires authentication, enter the name
+of the user here.
+
+|Password |If the proxy server requires authentication, enter the
+password here.
+
+|No Proxy Host |If certain hosts must be contacted directly (without
+going through the proxy), you can specify their names here. You can even
+use `+*+` as a wildcard. Example: `+*jenkins-test*+``+.example.com+`
+
+|Test URL |Enter the *full URL* of a host to use for testing the proxy
+configuration. Example: http://jenkins-ci.org/
+|===
+
+To temporarily disable the proxy configuration, you can add a single
+wildcard character `+*+` on the first line. 
+Erasing the value of *Server* will _discard all values_ and disable 
+the use of a proxy server.
+
+[[JenkinsBehindProxy-Whichpluginssupportaproxyserver]]
+===== Which plugins support a proxy server?
+
+[cols=",,,,",]
+|===
+|*Plugin* |*Since version* |*Supports HTTPS?* |*Supports Exclusions?*
+|*Supports authentication?*
+
+|https://wiki.jenkins.io/display/JENKINS/Fortify+On+Demand+Uploader+Plugin[Fortify
+On Demand Uploader Plugin]
+|https://github.com/jenkinsci/fortify-on-demand-uploader-plugin/releases/tag/fortify-on-demand-uploader-1.08[1.08]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Git+Plugin[Git Plugin]
+|https://github.com/jenkinsci/git-plugin/releases/tag/git-2.1.0[2.1.0]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Azure+DevOps+and+Team+Foundation+Server+Plugin[Azure
+DevOps and Team Foundation Server Plugin]
+|https://github.com/jenkinsci/tfs-plugin/releases/tag/tfs-5.1.0[5.1.0]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Openid+Connect+Authentication+Plugin[Openid
+Connect Authentication Plugin]
+|https://github.com/jenkinsci/oic-auth-plugin/releases/tag/oic-auth-1.4[1.4]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|===
```

Comment:
```suggestion
```

Reply:
This one is an error from my side in #3204 and it is my fault.

The wiki page https://wiki.jenkins.io/display/JENKINS//JenkinsBehindProxy is in fact another topic.

"Proxy" is for Jenkins to reach web resources outside, while "Reverse Proxy" is for request from the outside to reach Jenkins.

So in fact, this section should be removed from here and deserve a separated page. I'm taking care of the tracking: you only have to remove this (use the suggestion from Github)

Reply:
+ do not forget to remove the associated images :)

Reply:
ok